src/containers/offerContainer/offerContainer.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import OfferCard from '~/components/offer-card/OfferCard'
+import { Grid } from '@mui/material'
+import { CardsViewTypes, OfferResponse } from '~/types'
+
+interface OfferContainerProps {
+  viewMode: CardsViewTypes
+  offerCards: OfferResponse[]
+}
+
+const OfferContainer: FC<OfferContainerProps> = ({ viewMode, offerCards }) => {
+  const onBookmarkClick = (id: string) => {
+    console.log(id)
+  }
+
+  const arrayOfCards = offerCards.map((el) => (
+    <Grid item key={el._id} md={4} sm={4} xs={2}>
+      <OfferCard offer={el} onBookmarkClick={() => onBookmarkClick(el._id)} />
+    </Grid>
+  ))
+
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      {viewMode === 'grid' ? (
```

Comment:
as @Yurenko already mentioned, it would be good to use CardsViewEnums.Grid instead of 'grid'

Reply:
Fixed.